x-pack/plugins/osquery/public/packs/queries/platform_checkbox_group_field.tsx

```diff
@@ -82,57 +81,63 @@ export const PlatformCheckBoxGroupField = ({
     []
   );
 
-  const { isInvalid, errorMessage } = getFieldValidityAndErrorMessage(field);
+  // const { isInvalid, errorMessage } = getFieldValidityAndErrorMessage(field);
```

Comment:
no error validation?

Reply:
will remove the commented line. The error handling is now based within react hooks form itself.